packages/create-app/templates/default-app/plugins/README.md

```diff
@@ -4,6 +4,6 @@ This is where your own plugins and their associated modules live, each in a
 separate folder of its own.
 
 If you want to create a new plugin here, go to your project root directory, run
-the command `yarn backstage-cli create`, and follow the on-screen instructions.
+the command `yarn new`, and follow the on-screen instructions.
```

Comment:
Should it be `yarn new --select plugin` to be more explicit? I don't know if they are different or not. 

Optionally you can also add a link to [the docs](https://backstage.io/docs/plugins/create-a-plugin/)?

Reply:
`yarn new --select plugin` is specifically for a front end plugin.  `yarn new` gives you an option to pick what type of plugin you want

![image](https://user-images.githubusercontent.com/289860/235484797-e992c1b3-c37d-408c-ad23-1a2c918837bd.png)

Reply:
Oooh never knew that and always went to the docs for the specific frontend and backend commands 🤦 all the more reason your changes help!